src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -69,7 +69,7 @@ services.Configure<AspNetCoreInstrumentationOptions>(options =>
             options.Filter = (req) =>
             {
                 // only collect telemetry about HTTP GET requests
-                return httpContext.Request.Method.Equals("GET");
+                return req.Request.Method.Equals("GET");
```

Comment:
I'd suggest to keep this httpContext itself, and change the `req` to `httpContext` in line 69.